pkg/virt-launcher/notify-client/client.go

```diff
@@ -120,49 +122,64 @@ func libvirtEventCallback(c cli.Connection, domain *api.Domain, event *libvirt.D
 			domain.ObjectMeta.UID = spec.Metadata.KubeVirt.UID
 		}
 
-		log.Log.Infof("kubevirt domain status: %v (%v):%v (%v)", domain.Status.Status, status, domain.Status.Reason, reason)
```

Comment:
Probably an accidental removal after the rebase?

Reply:
Yes, in final phase I was removing my debug logs and this one is collateral damage. Will return it there.